kasa/aestransport.py

```diff
@@ -125,19 +125,19 @@ async def client_post(self, url, params=None, data=None, json=None, headers=None
         return resp.status_code, response_data
 
     def _handle_response_error_code(self, resp_dict: dict, msg: str):
```

Comment:
It'd be a great idea to have unit tests for the error handling logic.

Reply:
Added